src/Passes/Source/Rules/OperandPrototype.cpp

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include "Rules/OperandPrototype.hpp"
+
+namespace microsoft
+{
+namespace quantum
+{
+
+    IOperandPrototype::~IOperandPrototype() = default;
+    bool IOperandPrototype::matchChildren(Value* value, Captures& captures) const
+    {
+        auto user = llvm::dyn_cast<llvm::User>(value);
+        if (!children_.empty())
+        {
+            if (user == nullptr)
```

Comment:
Consider placing the line 14 between 16 and 17.

Reply:
Done.